### PR TITLE
Feature/error button

### PR DESCRIPTION
**Pull request by Karlfunhouse**
Added errors when the max range box value < min range box value.  Also added an error when the user guesses a number outside of the set range with N/A

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -105,22 +114,43 @@
...
     disableButtons()
     gameResetButton.removeAttribute('disabled')
   }
 
+  function guessErrorMessage() {
+    console.log(maxNumber)
+    if (parseInt(challengerGuesses[0].value) > maxNumber ||
+        parseInt(challengerGuesses[0].value) < minNumber) {
+          guessOneDisplay.innerText = 'N/A';
+          if (numberOfGuesses !== 0) {
```
Comment (KmBarnett) [inline main.js]: We should make a guess even if it is not within range it still counts as a guess.

File: main.js [hunk 2]
```diff
@@ -105,22 +114,43 @@
...
+
   function checkGuesses() {
     numberOfGuesses += 2;
     checkGuess1();
     checkGuess2();
   }
 
   function checkGuess1() {
-    if (parseInt(challengerGuesses[0].value) < randomNumber) {
+      if (parseInt(challengerGuesses[0].value) < randomNumber) {
```
Comment (KmBarnett) [inline main.js:147]: we need to fix this indentation

File: main.js [hunk 3]
```diff
@@ -105,22 +114,43 @@
...
+        parseInt(challengerGuesses[0].value) < minNumber) {
+          guessOneDisplay.innerText = 'N/A';
+          if (numberOfGuesses !== 0) {
+              numberOfGuesses -= 1;
+          };
+    } else {
+      guessOneDisplay.innerText = challengerGuesses[0].value;
+    } if (parseInt(challengerGuesses[1].value) > maxNumber ||
+        parseInt(challengerGuesses[1].value) < minNumber){
+          guessTwoDisplay.innerText = 'N/A';
```
Comment (KmBarnett) [inline main.js]: After talking it over with the project manager lets just disable the submit button in the event that the user tries to guess out of the range

File: main.js [hunk 4]
```diff
@@ -105,22 +114,43 @@
...
+          if (numberOfGuesses !== 0) {
+              numberOfGuesses -= 1;
```
Comment (KmBarnett) [inline main.js]: This becomes unnecessary if we disable submission

File: main.js (hunk 1 above)
Comment (Karlfunhouse) [reply main.js]: I think we should just disable the submit button if it's outside the range so it doesn't count as a guess.

File: main.js (hunk 3 above)
Comment (Karlfunhouse) [reply main.js]: Ah, we already agree i just didn't see this comment yet.  Okay cool sounds good.

File: main.js (hunk 4 above)
Comment (Karlfunhouse) [reply main.js]: cut it!

**Pull request merged**